src/components/SinglePost.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const SinglePost = ({ post }) => (
+  <div className="p-2">
+    <div className="d-flex justify-content-between px-5 fw-bold mt-1">
+      <p>Blog Post</p>
+      <NavLink to="/" className="text-decoration-none text-white">
+        <i className="fas fa-arrow-left" />
+        {' '}
+        Back
+      </NavLink>
```

Comment:
- [ ] Following the given design specs is one of the best practices clients would like. The layout of the `back` element is not as given in the design specs. Give this a fix. 😄
![Screenshot from 2021-10-14 16-36-30](https://user-images.githubusercontent.com/65501535/137328387-717d3fae-068c-4d25-94ae-fe67d10ec732.png)
![Screenshot from 2021-10-14 16-37-18](https://user-images.githubusercontent.com/65501535/137328552-26569cb0-0d53-4d4e-9a29-fa1261f09c25.png)